multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/model/SupportedParameterTest.java

```diff
@@ -36,6 +36,11 @@ public class SupportedParameterTest {
                                                                 SupportedParameters.VCAP_SERVICES_ENV,
                                                                 SupportedParameters.USER_PROVIDED_ENV, SupportedParameters.NO_HOSTNAME);
 
+    private static final Set<String> ROLLED_BACK_PARAMETERS = Set.of(
```

Comment:
Maybe rename to "CURRENTLY_NON_SUPPORTED_PARAMETERS" or "PLANNED_FUTURE_PARAMETERS".

We can use such feature to track eventually other new parameters for new CF functionalities.

Reply:
Renamed